packages/ui-components/src/icon/labicon.tsx

```diff
@@ -549,10 +549,6 @@ export class LabIcon implements LabIcon.ILabIcon, VirtualElement.IRenderer {
       if (uuid) {
         svgElement.dataset.iconId = uuid;
       }
-
-      if (title) {
```

Comment:
You need to go one step further and delete `title` from the arguments passed into `_initSvg`